src/search/find-visits.js

```diff
@@ -75,42 +77,40 @@ export function findVisitsToPages({pagesResult}) {
      * Raj Pratim Bhattacharya gmail rajpratim1234@gmail.com
      */
     
-    var comp = new Date();
-    var sDate = comp.getTime() - 100*24*60*60*1000 //100 days old search
-    var eDate =  comp.getTime()
-    
+
+    var current_time = new moment().valueOf();
+    var startDate = current_time - 100*24*60*60*1000; //100 days old search
```

Comment:
@obsidianart , I will be adding a commit based on the recent commit by @Treora  in #76 for date-range picker to maintain the style. and easier evaluation.